### PR TITLE
Am schimbat ce ai zis ca e de schimbat nenea

### PR DESCRIPTION
**Pull request by ZakkyPatronu**

### CODE REVIEW

File: .config/alacritty/alacritty.toml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import = ["/home/zakky/.config/alacritty/themes/themes/vapor.toml"]
```
Comment (ZmoleCristian) [inline .config/alacritty/alacritty.toml:1]: hardcode = bad

File: .zshrc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+zstyle ':vcs_info:git:*' unstagedstr '  '
+zstyle ':vcs_info:git:*' check-for-changes true
+
+
+# Define prompt vars
+setopt PROMPT_SUBST
+prompt_top="%F{white}%K{blue} 󱞬 %f%k"
+prompt_bottom="%F{white}%K{blue} 󱞪 %f%k"
+prompt_host="%F{white}%K{blue}  % archlinux "
+#prompt_user="%F{white}%K{blue}🥷 Andrei Veneticu-Cercel%f%k"
```
Comment (ZmoleCristian) [inline .zshrc:51]: scos

File: .zshrc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+prompt_user="%F{white}%K{blue}  Zakky %f%k"
```
Comment (ZmoleCristian) [inline .zshrc:52]: dinamic

**Review by ZmoleCristian — CHANGES_REQUESTED** [review]
plz change

File: .zshrc (hunk 2 above)
Comment (ZakkyPatronu) [reply .zshrc]: am scos

**Review by ZakkyPatronu — COMMENTED** [review]
gata man

**Review by ZmoleCristian — APPROVED** [review]
multumesc

**Pull request merged**